src/widgets/scale/lv_scale.h

```diff
@@ -23,16 +23,51 @@ extern "C" {
  *      DEFINES
  *********************/
 
+/**Default value of total minor ticks. */
+#define LV_SCALE_TOTAL_TICK_COUNT_DEFAULT (10U)
+LV_EXPORT_CONST_INT(LV_SCALE_TOTAL_TICK_COUNT_DEFAULT);
+
+/**Default value of major tick every nth ticks. */
+#define LV_SCALE_MAJOR_TICK_EVERY_DEFAULT (5U)
+LV_EXPORT_CONST_INT(LV_SCALE_MAJOR_TICK_EVERY_DEFAULT);
+
+/**Default value of scale label enabled. */
+#define LV_SCALE_LABEL_ENABLED_DEFAULT (1U)
+LV_EXPORT_CONST_INT(LV_SCALE_LABEL_ENABLED_DEFAULT);
+
 /**********************
  *      TYPEDEFS
  **********************/
 
+/**
+ * Scale mode
+ */
+enum {
+    LV_SCALE_MODE_HORIZONTAL    = 0x00,
```

Comment:
Maybe we can specify it as `LV_SCALE_MODE_HORIZONTAL_TOP` and `LV_SCALE_MODE_HORIZONTAL_BOTTOM` to specify in which direction the labels should be added.